plugins/gateway-backend/package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "@backstage/plugin-gateway-backend",
+  "version": "1.0.0",
+  "backstage": {
+    "role": "backend-plugin",
+    "pluginId": "gateway",
+    "pluginPackages": [
+      "@backstage/plugin-gateway-backend"
+    ]
+  },
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/backstage",
+    "directory": "plugins/gateway-backend"
+  },
+  "license": "Apache-2.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "backstage-cli package build",
+    "clean": "backstage-cli package clean",
+    "lint": "backstage-cli package lint",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack",
+    "start": "backstage-cli package start",
+    "test": "backstage-cli package test"
+  },
+  "dependencies": {
+    "@backstage/backend-plugin-api": "workspace:^",
+    "@opentelemetry/api": "^1.9.0",
+    "@opentelemetry/core": "^2.0.0",
```

```suggestion
    "@opentelemetry/core": "^1.0.0",
```

this matches the rest of the stuff in the lockfile

---

I've upgraded the deps in `backstage/package.json` instead

---

But now you've tied your feature PR with that pretty large otel bump. If the library changed significantly and emits metrics differently in unexpected ways it might have been better to make that bump separately. Maybe it's ok but just noting.

---

Fixed by #29467